tests/e2e/bootstrap_test.go

```diff
@@ -86,6 +86,34 @@ var _ = Describe("E2E - Bootstrapping node", Label("bootstrap"), func() {
 			Expect(numberOfFile).To(BeNumerically(">=", 1))
 		})
 
+		By("Configuring emulated TPM if needed", func() {
+			// Set correct value for TPM emulation
+			value := "false"
+			if emulateTPM == "true" {
+				value = "true"
+			}
+
+			// Patch the yaml file
+			err := tools.Sed("emulate-tpm:.*", "emulate-tpm: "+value, emulatedTPMYaml)
```

Comment:
we should probably use go templating for this if we expect to expand this further.

Reply:
You mean something like this: https://blog.gopheracademy.com/advent-2017/using-go-templates?

Anyway yes, this is something we can consider. I have no idea on how it works, but that's a good way to learn.